src/components/dropdown/themes/light/dropdown-item.base.scss

```diff
@@ -0,0 +1,62 @@
+@use '../../../../styles/common/component';
+@use '../../../../styles/utilities' as *;
+
+:host {
+    @include type-category('body-2');
+
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    width: 100%;
+    white-space: nowrap;
+    position: relative;
+    cursor: pointer;
+    color: color(gray, 800);
+}
+
+:host(:hover),
+:host(:focus) {
```

Comment:
@SisIvanova We should rely on the `active` attribute instead of `:focus`. We are not focusing the items since the focus remains on the target element.